### PR TITLE
chore: Update service binding docs

### PR DESCRIPTION
**Pull request by deekshas8**
## Context

AI/gen-ai-hub-sdk-js-backlog#157.

Update README to include a note about AI SDK handling of ai core service binding

## Definition of Done

- [ ] Code is tested (Unit, E2E)
- [ ] Error handling created / updated & covered by the tests above
- [ ] Documentation updated
- [ ] (Optional) Aligned changes with the Java SDK
- [ ] (Optional) Release notes updated

### CODE REVIEW

File: packages/foundation-models/README.md [hunk 1]
```diff
@@ -28,12 +28,14 @@ $ npm install @sap-ai-sdk/foundation-models
...
-  - Use the [`DeploymentApi`](https://github.com/SAP/ai-sdk-js/blob/main/packages/ai-api/README.md#create-a-deployment) from `@sap-ai-sdk/ai-api` to deploy a model to SAP generative AI hub.
-    For more information, see [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-generative-ai-model-in-sap-ai-core).
-    Deployment can be set up for each model and model version, as well as a resource group intended for use with the generative AI hub.
-  - Once a deployment is complete, the model can be accessed via the `deploymentUrl`.
+- Configure the project with **Node.js v20 or higher** and **native ESM** support.
+- Ensure a deployed OpenAI model is available in the SAP Generative AI Hub.
+  - Use the [`DeploymentApi`](https://github.com/SAP/ai-sdk-js/blob/main/packages/ai-api/README.md#create-a-deployment) from `@sap-ai-sdk/ai-api` to deploy a model to the SAP Generative AI Hub. For more information, see [this guide](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-generative-ai-model-in-sap-ai-core). Deployment can be set up for each model and model version, as well as a resource group.
```
Comment (MatKuhr) [inline packages/foundation-models/README.md:33]:
```suggestion
  - Use the [`DeploymentApi`](https://github.com/SAP/ai-sdk-js/blob/main/packages/ai-api/README.md#create-a-deployment) from `@sap-ai-sdk/ai-api` to deploy a model to the SAP Generative AI Hub.
  For more information, see [this guide](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-generative-ai-model-in-sap-ai-core).
```

File: packages/foundation-models/README.md [hunk 2]
```diff
@@ -28,12 +28,14 @@ $ npm install @sap-ai-sdk/foundation-models
...
+  - Once a deployment is complete, access the model via the `deploymentUrl`.
+
+> **Accessing the AI Core Service via the SDK**:
+> The SDK automatically retrieves the `AI Core` service binding from the `VCAP_SERVICES` environment variable, along with the service token needed for authentication.
```
Comment (MatKuhr) [inline packages/foundation-models/README.md:37]:
```suggestion
> The SDK automatically retrieves the `AI Core` service credentials from the `VCAP_SERVICES` environment variable and resolves the access token needed for authentication.
```

**Review by MatKuhr — COMMENTED** [review]
otherwise lgtm

File: packages/ai-api/README.md [hunk 3]
```diff
@@ -48,7 +48,11 @@ To ensure compatibility and manage updates effectively, we strongly recommend us
 ## Prerequisites
 
 - [Enable the AI Core service in SAP BTP](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup).
-- Project configured with Node.js v20 or higher and native ESM support enabled.
+- Configure the project with **Node.js v20 or higher** and **native ESM** support.
```
Comment (jjtang1985) [inline packages/ai-api/README.md:51]: I can see many copy/pasted content for different packages :)

**Review by jjtang1985 — CHANGES_REQUESTED** [review]
LGTM in general.
Please consider adding the doc for the Launchpad scenario.

File: packages/foundation-models/README.md (hunk 1 above)
Comment (deekshas8) [reply packages/foundation-models/README.md:33]: Updated in all READMEs

File: packages/foundation-models/README.md (hunk 2 above)
Comment (deekshas8) [reply packages/foundation-models/README.md:37]: Updated in all READMEs

File: packages/foundation-models/README.md [hunk 4]
```diff
@@ -28,12 +28,14 @@ $ npm install @sap-ai-sdk/foundation-models
...
 
 - [Enable the AI Core service in SAP BTP](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup).
-- Project configured with Node.js v20 or higher and native ESM support enabled.
-- A deployed OpenAI model in SAP Generative AI hub.
-  - Use the [`DeploymentApi`](https://github.com/SAP/ai-sdk-js/blob/main/packages/ai-api/README.md#create-a-deployment) from `@sap-ai-sdk/ai-api` to deploy a model to SAP generative AI hub.
-    For more information, see [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/create-deployment-for-generative-ai-model-in-sap-ai-core).
-    Deployment can be set up for each model and model version, as well as a resource group intended for use with the generative AI hub.
-  - Once a deployment is complete, the model can be accessed via the `deploymentUrl`.
+- Configure the project with **Node.js v20 or higher** and **native ESM** support.
+- Ensure a deployed OpenAI model is available in the SAP Generative AI Hub.
```
[the remark this answers is not in the document]
Comment (deekshas8) [reply packages/foundation-models/README.md:32]: Done

**Review by jjtang1985 — APPROVED** [review]

**Pull request merged**